Menus/MetagameGUI.cpp

```diff
@@ -5989,8 +5989,10 @@ void MetagameGUI::UpdateScenesBox(bool sceneChanged)
         }
 
         // Write the description, and add the total defense investment in this place so far as a lil stat
-        std::snprintf(str, sizeof(str), "Total base investments here: %doz", (int)floorf(m_pSelectedScene->GetTotalInvestment()));
-        m_pSceneInfoLabel->SetText(m_pSelectedScene->GetDescription() + "\n" + string(str));
+		Vector sceneSizeMeters = m_pSelectedScene->GetDimensions() / c_PPM;
+		std::string sceneDimensions = "Site Dimensions: " + std::to_string(sceneSizeMeters.GetFloorIntX()) + " x " + std::to_string(sceneSizeMeters.GetFloorIntY()) + " meters";
+		std::string sceneInvestments = "Total base investments here: " + std::to_string(static_cast<int>(m_pSelectedScene->GetTotalInvestment())) + "oz";
+        m_pSceneInfoLabel->SetText(m_pSelectedScene->GetDescription() + "\n\n" + sceneDimensions + "\n\n" + sceneInvestments);
```

Comment:
I don't foresee problems, but have you done a quick launch of the metagame to make sure
1. It works fine?
2. It looks fine, especially with the rearranged stuff and extra spacing?

Reply:
It's fiiiiiinnnnneeeeeeeeeee.